src/html/split-sections.js

```diff
@@ -25,18 +32,25 @@ function split({ content }) {
   // include the very start and end of the document
   const starts = [0, ...dividers];
   const ends = [...dividers, mdast.children.length];
-  content.sections = _.zip(starts, ends)
+
+  content.mdast.children = _.zip(starts, ends)
   // but filter out empty section
     .filter(([start, end]) => start !== end)
   // then return all nodes that are in between
     .map(([start, end]) => {
     // skip 'thematicBreak' nodes
       const index = mdast.children[start].type === 'thematicBreak' ? start + 1 : start;
-      return {
-        type: 'root',
+      const section = {
+        type: 'section',
```

Comment:
New node type is a good idea.

Reply:
Is it ok to just inline the node type or do we have a central place to expose them as enum constants that I could use?

Reply:
maybe a schema would be in order?

Reply:
@rofe I added a schema, and updated existing ones accordingly with the changes.
Was just wondering if we have a sort of global "constants" file where I would be pulling the string from, or if it is ok to just inline for now